batch/batch/front_end/front_end.py

```diff
@@ -1063,6 +1064,10 @@ async def _create_jobs(userdata: dict, job_specs: dict, batch_id: int, update_id
         if cloud == 'azure' and all(envvar['name'] != 'AZURE_APPLICATION_CREDENTIALS' for envvar in spec['env']):
             spec['env'].append({'name': 'AZURE_APPLICATION_CREDENTIALS', 'value': '/gsa-key/key.json'})
 
+        cloudfuse = spec.get('gcsfuse') or spec.get('cloudfuse')
+        if not is_developer and user not in ('ci', 'test', 'test-dev') and cloudfuse is not None:
```

Comment:
Should just need this

```suggestion
        if not is_developer and user not in ('ci', 'test', 'test-dev') and cloudfuse is not None and len(cloudfuse) > 0:
```